### ISSUE TITLE
Schema error in web socket connection for trips

### ISSUE THREAD

**Issue by ahknight** [armills/aioautomatic#3]
* aioautomatic version:0.6.2
* Python version:3.4.2
* Operating System:Raspbian Jesse

### Description

Using with Home Assistant.  HASS logs show aioautomatic threw a schema error in the logs after authenticating with Automatic.

### What I Did

1. Setup client/secret.
2. OAuth login.
3. View logs.

```
Message does not match schema: {'vehicle': 'https://api.automatic.com/vehicle/XXXX/', 'end_address': None, 'driver': 'https://api.automatic.com/user/XXXX/', 'tags': [], 'end_timezone': 'America/Chicago', 'start_timezone': 'America/Chicago', 'started_at': '2016-08-06T23:09:57.500000Z', 'idling_time_s': 0, 'distance_m': 23.9, 'start_location': {'accuracy_m': 3.6, 'lat': dd.ddddd, 'lon': -dd.ddddd}, 'end_location': {'accuracy_m': 65.0, 'lat': dd.ddddd, 'lon': -dd.ddddd}, 'id': 'T_d9fbaa2f0d284ab7', 'start_address': None, 'url': 'https://api.automatic.com/trip/XXXX/', 'duration_s': 42.1, 'path': None, 'user': 'https://api.automatic.com/user/XXXX/', 'ended_at': '2016-08-06T23:10:39.600000Z'}
```

**Comment by ahknight** [comment]
Perhaps page and limit should be marked optional? I have two vehicles in there so it may not always show those.

**Comment by armills** [comment]
Page and limit are only in the request schema, not in the response, and they're optional.

Is this an actual event, or a simulated event from Automatic's web interface? Some of their simulated events don't match the actual events they send.

Can you enable debug logging to see what part of the schema is failing? In your hass configuration, add the following and post the whole error.

```yaml
logger:
  default: info
  logs:
    homeassistant.components.device_tracker.automatic: debug
    aioautomatic: debug
```

**Comment by ahknight** [comment]
20:26:08 DEBUG (MainThread) [aioautomatic.base] Received {'_metadata': {'count': 2416, 'previous': None, 'next': 'https://api.automatic.com/trip/?started_at__lte=1505420766980&started_at__gte=1398634360464&cursor=4b6c56af-48d4-42fa-b4d2-7207a30b3526&limit=1&vehicle=C_XXXXXXXXXX1a667d&page=2'}, 'results': [{'end_address': None, 'url': 'https://api.automatic.com/trip/T_XXXXXXXXXXXX4ab7/', 'start_address': None, 'vehicle': 'https://api.automatic.com/vehicle/C_XXXXXXXXXX1a667d/', 'idling_time_s': 0, 'user': 'https://api.automatic.com/user/U_XXXXXXXXXXXX5d8b/', 'path': None, 'end_timezone': 'America/Chicago', 'start_location': {'lon': -9X.X8613, 'accuracy_m': 3.6, 'lat': 3X.X1695}, 'duration_s': 42.1, 'tags': [], 'driver': 'https://api.automatic.com/user/U_XXXXXXXXXXXX5d8b/', 'end_location': {'lon': -9X.X3057, 'accuracy_m': 65.0, 'lat': 3X.X6402}, 'id': 'T_XXXXXXXXXXXX4ab7', 'started_at': '2016-08-06T23:09:57.500000Z', 'ended_at': '2016-08-06T23:10:39.600000Z', 'start_timezone': 'America/Chicago', 'distance_m': 23.9}]}

20:26:08 ERROR (MainThread) [homeassistant.components.device_tracker.automatic] Message does not match schema: {'end_address': None, 'url': 'https://api.automatic.com/trip/T_XXXXXXXXXXXX4ab7/', 'start_address': None, 'vehicle': 'https://api.automatic.com/vehicle/C_XXXXXXXXXX1a667d/', 'idling_time_s': 0, 'user': 'https://api.automatic.com/user/U_XXXXXXXXXXXX5d8b/', 'path': None, 'end_timezone': 'America/Chicago', 'start_location': {'lon': -9X.X8613, 'accuracy_m': 3.6, 'lat': 3X.X1695}, 'duration_s': 42.1, 'tags': [], 'driver': 'https://api.automatic.com/user/U_XXXXXXXXXXXX5d8b/', 'end_location': {'lon': -9X.X3057, 'accuracy_m': 65.0, 'lat': 3X.X6402}, 'id': 'T_XXXXXXXXXXXX4ab7', 'started_at': '2016-08-06T23:09:57.500000Z', 'ended_at': '2016-08-06T23:10:39.600000Z', 'start_timezone': 'America/Chicago', 'distance_m': 23.9}

**Comment by armills** [comment]
It looks like the address isn't being sent with those responses. I've updated the schema and shipped version 0.6.3. It will be included with Home Assistant version 0.54. Thanks for the report!
